### DISCUSSION TITLE
Renovate is timing out as of February 3rd, 9AM EST

### DISCUSSION THREAD

**Discussion by esetnik** [renovatebot/renovate#20211] [Request Help]
### I'm using the Mend Renovate hosted app.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe your problem.

Originally reported as https://github.com/renovatebot/renovate/issues/20205

![image](https://user-images.githubusercontent.com/664434/216708262-226a72fc-e1b4-4a08-aa5b-3eaf1b6ac617.png)

### Relevant debug logs.

<details><summary>Logs</summary>

```
DEBUG: Using cached etag
DEBUG: Using cached etag
DEBUG: Cached npm result is revalidated
{
  "packageName": "saml"
}
DEBUG: Cached npm result is revalidated
{
  "packageName": "yarnhook"
}
WARN: Job timeout
```

</details>

**Comment by rarkins** [comment]
How long did previous jobs take?

**Reply by esetnik** [reply]
Previous successful run was:
```
INFO: Repository finished
{
  "cloned": true,
  "durationMs": 239611
}
```

**Comment by rarkins** [comment]
This is the last commit/change in the hosted app: 
<img width="182" alt="image" src="https://user-images.githubusercontent.com/6311784/216758705-05da9bd2-6db8-44c7-b556-5667d5387f4c.png">

**Comment by esetnik** [comment]
Suddenly and randomly started working again with no code changes on my side:

<img width="1245" alt="image" src="https://user-images.githubusercontent.com/664434/216789052-ad4e121d-cffe-47ae-8fe2-37adc4a377bd.png">

**Reply by esetnik** [reply]
Next run duration (#998343891) was:
```
INFO: Repository finished
{
  "cloned": true,
  "durationMs": 322362
}
```
